### PR TITLE
Implement the user command for bakery owners to check pastry stock

### PR DESCRIPTION
**Pull request by calebchongsj**
Bakery owners need to check if they have sufficient pastries to sell to customers.

### CODE REVIEW

**Comment by yikyak02** [comment]
lgtm!

**Review by ryenl — COMMENTED** [review]
Good use of assertions. LGTM!

**Review by bbryant824 — COMMENTED** [review]
Good use of Keywords.

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 1]
```diff
@@ -157,11 +161,31 @@ public void addIngredient(Ingredient ingredient) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public void addCustomerOrder(CustomerOrder order) {
+            throw new AssertionError("This method should not be called.");
```
Comment (bbryant824) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:166]: good use of assertions here.

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 2]
```diff
@@ -157,11 +161,31 @@ public void addIngredient(Ingredient ingredient) {
...
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public OrderList getOrderList() {
+            throw new AssertionError("This method should not be called.");
+        }
+
+        @Override
+        public Inventory getInventory() {
```
Comment (bbryant824) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:185]: looks good to me

**Review by bbryant824 — COMMENTED** [review]
Good implementation. Well done Caleb!

**Pull request merged**